### DISCUSSION TITLE
Failed to initialize Shoutrrr notifications: error initializing router services: unknown service \"matrix

### DISCUSSION THREAD

**Discussion by 13hannes11** [containrrr/watchtower#1058] [Q&A] [answered]
Hi,
I am trying to set up shoutrrr notifications and get the following error message.

```
Failed to initialize Shoutrrr notifications: error initializing router services: unknown service \"matrix
```

Is watchtower not yet using the most recent shoutrrr version? Or am I doing something else wrong.

I'm using the following environment:
```
WATCHTOWER_NOTIFICATION_URL=matrix://${MATRIX_USER}:${MATRIX_USER_PW}@matrix.org/?rooms=${MATRIX_ROOM_ID}
```

Thanks in advance!

**Comment by simonaronsson** [comment, marked as answer]
Hi,

The latest watchtower release was in April, while Matrix was added to Shoutrrr in May. A new watchtower release will be cut soon ™️ , but until then, matrix won't work.

Best,
Simme
